frontend/src/components/Workspace/FlowChart/Dialog/FileSelectDialog.tsx

```diff
@@ -233,6 +262,8 @@ const TreeNode = memo(function TreeNode({
         label={
           multiSelect && node.nodes.filter((node) => !node.isDir).length > 0 ? (
             <TreeItemLabel
+              fileType={fileType}
+              shape={node.shape}
```

Comment:
Apologies, I've found directories show error icons unless the files in the directory does not have errors.

![Screenshot 2024-01-09 at 16 25 21](https://github.com/arayabrain/barebone-studio/assets/42664619/f830d3cf-4d8f-4bbf-8591-2475ab219f51)

Could you fix not to show status, shape and update shape icons for directories?

Reply:
@ReiHashimoto Sorry, can I ask for data in a folder like yours to check again?

Reply:
@sanglevinh 
Please use this file. Extracting the file on `OPTINIST_DIR/input` would work.
https://drive.google.com/file/d/1pfgpKs-1jyHy6vAaSuAY4BfAyXjBOcMN/view?usp=drive_link

Reply:
@ReiHashimoto Sorry, can you write down each step to reproduce this error? I unzipped your document and it comes out like the image below, but as far as I can see, uploading images is only done for files with the extension .tiff or .tif so I can't reproduce it like yours
![Screenshot 2024-01-16 at 14 13 03](https://github.com/arayabrain/barebone-studio/assets/109060170/658afa00-ded9-4221-aaa2-b46f07bf29ee)

Reply:
@sanglevinh 
OK, 
1. checkout to this repository
    ```
    git switch feature/check_format_tiff_file
    ```
2. merge BE branch
    ```
    git merge origin/feature/check-tiff-format-when-upload
    ```
3. extract the data in OPTINIST_DIR/input
    OPTINIST_DIR would be following structure after this.
    ![Screenshot 2024-01-16 at 16 37 01](https://github.com/arayabrain/barebone-studio/assets/42664619/fef2b6f6-d7a0-4166-83eb-15ab80e6b61b)
4. run optinist BE and FE
    ```
    python main.py --reload
    ```
    ```
    cd frontend
    yarn start
    ```
5. On WORKFLOW tab window, add image data node(Nodes > Data > image at the left sidebar).
6. Click "select from uploaded files" button.
7. Image's shape are displayed like this.
    ![Screenshot 2024-01-16 at 16 41 37](https://github.com/arayabrain/barebone-studio/assets/42664619/b0544501-ef28-43df-ab1c-b158d7e932af)

I hope it would help you.

Reply:
@ReiHashimoto I have fixed it, please check to see if it is correct

Reply:
@sanglevinh 
Thank you.
One more thing, BE does not support specifying a directory and batch update the shapes of the files in that directory.
So, could you remove update icon of the directories, too?

<img width="648" alt="Screenshot 2024-01-16 at 19 26 57" src="https://github.com/arayabrain/barebone-studio/assets/42664619/90a9efd4-9718-42bd-8a82-67f30d1cd075">

Reply:
@ReiHashimoto i fixed it, pls check it again